src/ncp/ncp_base_radio.cpp

```diff
@@ -123,15 +124,32 @@ otError NcpBase::PackRadioFrame(otRadioFrame *aFrame, otError aError)
     return error;
 }
 
-void NcpBase::LinkRawReceiveDone(otInstance *, otRadioFrame *aFrame, otError aError)
+#if OPENTHREAD_CONFIG_MULTIPAN_RCP_ENABLE
+void NcpBase::NotifySwitchoverDone(otInstance *aInstance, bool aSuccess)
 {
-    sNcpInstance->LinkRawReceiveDone(aFrame, aError);
+    OT_UNUSED_VARIABLE(aInstance);
+    NotifySwitchoverDone(aSuccess);
 }
 
-void NcpBase::LinkRawReceiveDone(otRadioFrame *aFrame, otError aError)
+void NcpBase::NotifySwitchoverDone(bool aSuccess)
 {
-    uint8_t header = SPINEL_HEADER_FLAG | SPINEL_HEADER_IID_0;
+    uint8_t         header = SPINEL_HEADER_FLAG | SPINEL_HEADER_IID_BROADCAST;
```

Comment:
Should `SPINEL_HEADER_TX_NOTIFICATION_IID` be more appropriate to use instead of SPINEL_HEADER_IID_BROADCAST? I know at the end, it is derived from the same macro.